Updated composer.json and DatabaseAnalyzer.php

theStormWinter opened the pull request:
Updated for using nette/php-generator up to version 3.2.3 
Repaired in DatabazeAnalyzer.php array key "Extra" to lowercase

theStormWinter committed:
Updated for using nette/php-generator up to version 3.2.3
Repaired in DatabazeAnalyzer.php array key "Extra" to lowercase

mabar commented on composer.json:
```diff
@@ -13,7 +13,7 @@
     "php": ">=7.1",
     "nette/database": "^2.4.0",
-    "nette/php-generator": "^2.5|3.1",
+    "nette/php-generator": "^2.5|3.2.3",
```
`^2.5.0|^3.0.0`, exact 3.x version is a bug

mabar commented on src/Analyser/Database/DatabaseAnalyser.php:
```diff
@@ -70,7 +70,7 @@ protected function analyseColumns(Table $table): void
 			$column->setType(Helpers::columnType($col['nativetype']));
 			$column->setDefault($col['default']);
-			$column->setOnUpdate(Strings::contains($col['vendor']['Extra'], 'on update'));
+			$column->setOnUpdate(Strings::contains($col['vendor']['extra'], 'on update'));
```
Capitalized key was introduced by #16, someone should check proper behavior before any changes of that https://github.com/contributte/nextras-orm-generator/commit/b0002a1da7578586a83576d341e224567eaf79be

f3l1x commented:
ping @theStormWinter

theStormWinter committed:
Updated dependency to nette/php-generator up to version ^3.2
Added in DatabazeAnalyzer.php array key "Extra" or "extra" and "Type" or "type" check

Zazimou committed:
Update dependencies to Nette 3

f3l1x commented:
Merged in #29
